index.html

```diff
@@ -343,7 +343,7 @@ <h3>Information for Authors</h3>
 	    <h3>Information for User Agents</h3>
 	    <p>The user agent MUST do the following to implement <pref>aria-activedescendant</pref>:</p>
 	    <ol>
-	      <li>Implement the host language method for keyboard navigation so that widgets that support <code>aria-activedescendant</code> may be included in the tab order.</li>
+	      <li>Implement the host language method for keyboard navigation so that widgets that support <code>aria-activedescendant</code> MAY be included in the tab order.</li>
```

Comment:
I think this one should be "can"?

Reply:
```suggestion
	      <li>Implement the host language method for keyboard navigation so that widgets that support <code>aria-activedescendant</code> can be included in the tab order.</li>
```